tests/scripts/generate_psa_tests.py

```diff
@@ -614,19 +614,23 @@ def all_keys_for_usage_extension(self) -> List[StorageKey]:
         keys = [] #type: List[StorageKey]
         prev_builder = self.key_builder
 
-        # Generate the key without usage extension
+        # Generate the keys without usage extension
         self.key_builder = StorageKeyBuilder(usage_extension=False)
         alg_with_keys = self.gather_key_types_for_sign_alg()
-        key_restrictions = StorageKey.EXTENDABLE_USAGE_FLAGS_KEY_RESTRICTION
+        key_filter = StorageKey.EXTENDABLE_USAGE_FLAGS_KEY_RESTRICTION
+
+        # Use a lookup for the extendable usage flags to able to sort them
+        usage_lookup = {} #type: Dict[str, psa_storage.Expr]
+        for usage_flag in StorageKey.EXTENDABLE_USAGE_FLAGS:
+            usage_lookup[usage_flag.string] = usage_flag
+
         # Walk through all combintion. The key types must be filtered to fit
         # the specific usage flag.
         keys += [key
-                 for usage in StorageKey.EXTENDABLE_USAGE_FLAGS
+                 for usage in sorted(usage_lookup)
```

Comment:
“Fix generated file modification” seems to be doing a lot of extra work. Isn't this just `sorted(EXTENDABLE_USAGE_FLAGS, key=str)`?